packages/memprocfs.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,7 +4,7 @@ Import-Module vm.common -Force -DisableNameChecking
 $toolName = 'MemProcFS'
 $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
 
-$zipUrl = 'https://github.com/ufrisk/MemProcFS/releases/download/v5.14/MemProcFS_files_and_binaries_v5.14.11-win_x64-20250501.zip'
-$zipSha256 = '40ce4ac604933dd919444a9cee42431b3fe43813fa4155bef17466b7db03d764'
+$zipUrl = 'https://github.com/ufrisk/MemProcFS/releases/download/v5.14/MemProcFS_files_and_binaries_v5.14.13-win_x64-20250520.zip'
```

Comment:
The URL have changed again. We have the same issue in https://github.com/mandiant/VM-Packages/pull/1420.

We need to either downgrade to v5.13.4 (as the asset of that version doesn't seem to change anymore) or the help of MemProcFS by changing how the releases are done. I have sent an issue to see if they are able to help: https://github.com/ufrisk/MemProcFS/issues/389